CHANGELOG.md

```diff
@@ -1,14 +1,52 @@
 # Changelog Commonalities
 
 ## Table of Contents
+- **[r2.1](#r21)**
+- **[v0.4.0](#v040)**
+- **[v0.4.0-rc.2](#v040-rc2)**
+- **[v0.4.0-rc.1](#v040-rc1)**
+- **[v0.4.0-alpha.1](#v040-alpha1)**
+- **[v0.3.0](#v030)**
+- **[v0.2.0](#v020)**
+- **[v0.1.0 - Initial version](#v010---initial-version)**
 
-- [v0.4.0](#v040)
-- [v0.4.0-rc.2](#v040-rc2)
-- [v0.4.0-rc.1](#v040-rc1)
-- [v0.4.0-alpha.1](#v040-alpha1)
-- [v0.3.0](#v030)
-- [v0.2.0](#v020)
-- [v0.1.0 - Initial version](#v010---initial-version)
+
+# r2.1
+## Release Notes
+
+This release contains documents and artifacts of Commonalities version 0.5.0-alpha.1:
+* Commonalities approved deliverables in **[documentation](https://github.com/camaraproject/Commonalities/tree/r2.1/documentation)** folder.
+* Commonalities approved artifacts in **[artifacts](https://github.com/camaraproject/Commonalities/tree/r2.1/artifacts)** folder.
+
+**The relevant details of authentication and consent collection are covered by [release 2.1](https://github.com/camaraproject/IdentityAndConsentManagement/releases) of Identity and Consent Working Group documents.**
+
+### Added
+* Common 'area' data-type added to CAMARA_common.yaml by @tlohmar in https://github.com/camaraproject/Commonalities/pull/315
+* Security and Privacy Considerations for Filtering in API Design Guidelines by @rartych in https://github.com/camaraproject/Commonalities/pull/331
+* Security scheme added to CAMARA_common.yaml by @rartych in https://github.com/camaraproject/Commonalities/pull/335
+* VERSION.yaml file added to indicate Commonlities versionby @rartych in https://github.com/camaraproject/Commonalities/pull/339
+* Filtering for boolean guideline and examples in API Design Guidelines by @rartych in https://github.com/camaraproject/Commonalities/pull/336
+* Guidelines on the coverage of error codes in API-Testing-Guidelines by @jlurien in https://github.com/camaraproject/Commonalities/pull/343
+
+### Changed
+* Normalization of error status and code allowed values using `enum` by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/316
+* Guidelines for subscription and event notification in API Design Guideliness 0.5.0 by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
+  - updated `terminationReason` in event notification type "subscription-ends"
+  - updated description of `sink` and `sinkCredential` attributes for subscription
+  - added rules for subscriptions with device identifier attribute
+  - added section 11.7 Resource access restriction relevant to subscriptions
+  - added clarification on `expiresAt` attribute for subscription
+* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324
```

Comment:
Suggestion:

* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324 and https://github.com/camaraproject/Commonalities/pull/346

Reply:
resolved in https://github.com/camaraproject/Commonalities/pull/342/commits/acfbc56e8025ea10b4294f310a79637c2ecd0503